src/main/java/net/minecraftforge/items/wrapper/CombinedInvWrapper.java

```diff
@@ -126,4 +135,13 @@ public boolean isItemValid(int slot, @NotNull ItemStack stack)
         int localSlot = getSlotFromIndex(slot, index);
         return handler.isItemValid(localSlot, stack);
     }
+
+    @Override
+    public IOGuarantees getIOGuarantees()
```

Comment:
This implementation is wrong if some of the inventories are linked, which can't really be tested for.